partner_address_street3/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-tools https://github.com/akretion/server-tools 14-mig-base_view_inheritance_extension
```

Comment:
this has been merged
you can simplify it by pinning oca/14.0 branch

Reply:
actually we should drop the file, we have MQT=PIP